client/src/styles/PivotTableConfigMenu.scss

```diff
@@ -0,0 +1,5 @@
+.PivotTableConfigMenu {
+  .totalToggle {
+    padding: 0.5rem 0;
+  }
+}
```

Comment:
No newline at the end of this file ^^
Not sure if we consider this an issue, but maybe add one?